buddy_mentorship/templates/buddy_mentorship/home.html

```diff
@@ -6,13 +6,78 @@
 
 {% block content %}
   <div class="starter-template">
-    Tada! It's the homepage!<br>
-
-    {% if user.is_authenticated %}
-    	<p>you are logged in</p>
-	{% else %}
-	    <p>you are not logged in</p>
-	{% endif %}
+  		<h1 class="display-3 text-center">
+  			Welcome to <span class="text-primary">ChiPyBuddies</span>.
+  		</h1>
+  		<p class="lead text-center text-muted mb-6 mb-lg-8">
+  			Find Python mentors or mentees in the Chicago-land area.
```

Comment:
"Chicagoland" might be more commonly in use than "Chicago-land". Don't feel strongly.